dev/archery/archery/integration/datagen.py

```diff
@@ -1891,9 +1887,11 @@ def _temp_path():
 
         generate_duration_case(),
 
-        generate_interval_case(),
+        generate_interval_case()
+        .skip_tester('JS'),
 
-        generate_month_day_nano_interval_case(),
+        generate_month_day_nano_interval_case()
+        .skip_tester('JS'),
```

Comment:
Why skip this test? Isn't this the point of this pull request?

Reply:
Hi @domoritz The issues about interval data in JSON format just been fixed in the latest commit. Please check it. Thank you